src/sender.h

```diff
@@ -56,6 +56,12 @@ class SMTP_EXPORT Sender : public QObject
         TlsConnection       // STARTTLS
     };
 
+    enum PeerVerificationType
+    {
+        VerifyNone,
+        VerifyPeer
+    };
```

Comment:
Please also add Q_ENUM, which would make it dependent on Qt 5.5

Reply:
Since Q_ENUM now makes it depend on Qt 5.5 the Readme may should be changed to read something like 
... library writen for Qt 5 (Qt version > 5.5 with C++11) ...